### PR TITLE
Cleaning backend logic

### PR DESCRIPTION
**Pull request by tweekes1**
Removed code that was unused.

### CODE REVIEW

File: api/app/__init__.py [hunk 1]
```diff
@@ -1,19 +1,9 @@
...
 from app import routes
 
-app.config.from_object('config')
-"""
-====================================================================================
-                            VARS NEEDED FOR AUTH AND FIRESTORE
-cors = CORS(app, resources={r"/*": {"origins": "*"}})
-cred = credentials.Certificate('key.json')
-firebase_app = initialize_app(cred)
-====================================================================================
```
Comment (ShinValor) [inline api/app/__init__.py:17]: This is needed for firebase jwt

File: api/app/routes.py [hunk 2]
```diff
@@ -7,17 +7,8 @@
 import os, sys, json
 import jsonpickle as jp
 
-from app import app
-"""
-====================================================================================
-                            VARS NEEDED FOR AUTH AND FIRESTORE
-vars from the app.__init__ file that contains vars needed for auth and firestore
-from app import cors, cred, firebase_app
-from firebase_admin import auth, firestore
```
Comment (ShinValor) [inline api/app/routes.py:16]: This is needed for firebase jwt

File: api/app/__init__.py (hunk 1 above)
Comment (tweekes1) [reply api/app/__init__.py:17]: I know that but @junqili259 adds the vars in the routes module directly. The code being remove is commented and never actually executed.

Comment (ShinValor) [reply api/app/__init__.py:17]: Ok

**Pull request merged**